### ISSUE TITLE
users should be able to follow compilation process

### ISSUE THREAD

**Issue by rezib** [edf-hpc/unclebench#2]
On the first run, unclebench compiles the benchmark, with the following output:

```
$ ubench run -b vasp -p athos

---- vasp description files are already present in run directory and will be overwritten.

---- launching  vasp  with default node configuration
```

Then, the compilation process starts, the command ends and the user has no hint to follow the compilation. So there is no way to check if everything is going well, and when the job start...

It would be better if the user has a way to know when the compilation happens, if everything is going well, where to check for errors, and when he/she's supposed to get jobs running.

**Comment by pierretr** [comment]
This issue could be solved by a "ubench log" command which would return logs from each step of the benchmark. Thus user would be able to follow all the steps from compilation to the end of the benchmark execution.

**Comment by pierretr** [comment]
Available since version 0.2.0.
